pkg/apis/capabilities/v1alpha1/zz_generated.openapi.go

```diff
@@ -42,6 +40,7 @@ func schema_pkg_apis_capabilities_v1alpha1_API(ref common.ReferenceCallback) com
 		Schema: spec.Schema{
 			SchemaProps: spec.SchemaProps{
 				Description: "API is the Schema for the apis API",
+				Type:        []string{"object"},
```

Comment:
there should be no changes in `pkg/apis/capabilities/v1alpha1/zz_generated.openapi.go`

My guess? you are using a diferent version of the operator-sdk. Our requirements state that:
```
operator-sdk version v0.8.0
```

There is WIP PR to upgrade operator-sdk #310 to 0.14.1